src/components/fx/hover.js

```diff
@@ -339,7 +340,8 @@ function _hover(gd, evt, subplot, noHoverEvent) {
 
             xpx = evt.clientX - dbb.left;
             ypx = evt.clientY - dbb.top;
-
+            
+            plotApi.recalculateTransformInverseIfNecessary(gd);
```

Comment:
I am not sure we we could attach this function to the `Plotly.js` API.
Instead you could attach it to `gd._fullLayout._recalcTrans...` and call it during interactions.

Reply:
I did wonder about that. Will do, thanks :)